.github/workflows/lintcommits.yml

```diff
@@ -15,6 +15,9 @@ jobs:
         uses: actions/github-script@v6
         with:
           script: |
+            const excluded_bot_ids = {
```

Comment:
camelCase for JS code pls

Reply:
Got confused by the `pull_request` nonsense here. Fixed!